org.cvfosammmm.Setzer.json

```diff
@@ -1,7 +1,7 @@
 {
     "app-id": "org.cvfosammmm.Setzer",
     "runtime": "org.gnome.Platform",
-    "runtime-version": "3.38",
+    "runtime-version": "40",
```

Comment:
can't you move to 41 instead?

Reply:
I started with 41, then I thought that moving progressively would have been better.
I'll send a new PR.